src/components/PlaySongs/styles.ts

```diff
@@ -63,19 +68,27 @@ export const Title = styled.div`
     overflow: hidden;
     text-overflow: ellipsis;  
     align-self: center; 
-    color: ${({ theme }) => theme.colors.blueGreen};
+    color: ${({ theme }) => theme.colors.whiteFontColor};
     font-size: 1.2rem;
     line-height: 1.6rem;
     margin-bottom: 15px;
-    &:hover {
-        overflow: visible;
-    }
 `
+
 export const Button = styled.button`
+    border: 0;
+    background-color: inherit;
+    cursor: pointer;
+    transition: 0.3s ease-in-out;
+    &:hover {
+        transform: scale(1.1);
+    }
 `
+
 export const ButtonCanBeDisabled = styled.button<{disabled: boolean}>`
+    
     svg {
-        color: ${({ theme, disabled }) => disabled === true? "grey": theme.colors.blue};
+        color: ${({ theme, disabled }) => disabled === true ? theme.colors.disabled: theme.colors.blue};
+        cursor: ${({ theme, disabled }) => disabled === true ? "not-allowed" : "pointer"};
```

Comment:
I would personally prefer not adding this 'not-allowed' value here, as it looks a bit strange to me. The changing colours already convey the message. But it is up to you.

Reply:
i want to imitate the standard button disabled property